superset/assets/src/query/Metric.ts

```diff
@@ -0,0 +1,95 @@
+import Column from './Column';
+import FormData from './FormData';
+
+export enum MetricKey {
+  METRIC = 'metric',
```

Comment:
You mean making it `SECONDARY_METRIC = 'SECONDARY_METRIC'`?
Yeah, we can. Just need to change the values hardcoded on the server side. It is currently expecting lower_sneak_case like `secondary_metric`.

Reply:
hrm, I'm thinking maybe we should leave the values as `lower_snake_case` as they're used in the form_data json (example below). It may be odd to have `UPPER_CASE` keys in the json. Thoughts?

```
    "secondary_metric": {
      "type": "SelectControl",
      "label": "Color Metric",
      "default": null,
      "description": "A metric to use for color"
    }
```

Reply:
ah that makes sense, I'm okay with it for that reason but can we add a comment explaining why they keys/values have different casing?